src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. Here you have a right margin on all `.stars__star` elements. However, you're trying to remove the margin from the last child which is unnecessary if you only add margin to the left side of each star except the first child. Consider using `.stars__star:not(:first-child)` with a `margin-left` to ensure only spacing between the stars without having to reset the margin on the last child.